src/keyboard-x11.c

```diff
@@ -292,6 +255,53 @@ Keyboard* keyboard_new()
 static void keyboard_init(Keyboard *self)
 {
     self->pPrivate = keyboard_get_instance_private(self);
+    self->pPrivate->lLayouts = g_hash_table_new_full(g_str_hash, g_str_equal, NULL, freeLayout);
+
+    // Read all available layouts
+    struct rxkb_context *pContext = rxkb_context_new(RXKB_CONTEXT_LOAD_EXOTIC_RULES);
+    g_assert(pContext);
+
+    if (!rxkb_context_include_path_append_default(pContext))
+    {
+        g_warning("Failed to include default paths");
+    }
+
+    if (!rxkb_context_parse(pContext, "evdev"))
+    {
+        g_warning("Failed to parse XKB descriptions");
+    }
+
+    struct rxkb_layout *pRxkbLayout = rxkb_layout_first(pContext);
+    g_assert(pRxkbLayout);
```

Comment:
This g_assert call should either be dropped or accompanied by a -DG_DISABLE_ASSERT in the compiler options.

Reply:
Done.